src/settings/BatchGroupConfigurationSettings/utils.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+  keys,
+  pickBy,
+} from 'lodash';
+
+const REGEXP_URI = new RegExp('^$|([f][t][p])([s])?://.+$');
+
+export const validateUploadURI = (value) => {
+  const isTrue = REGEXP_URI.test(value);
```

Comment:
`isTrue` is a bad naming. You can't say what isTrue and what is not. Please give a proper name

Reply:
Renamed it